package sk;

import "k-clustering.proto";

message SKModelConfig {
    required uint64 img_w = 1;
    required uint64 img_h = 2;
    required uint64 reg_w = 3;
    required uint64 reg_h = 4;
    optional KClusteringConfig clustering = 5;
    optional double thres = 6;
    repeated uint64 total_counter = 7;
    repeated uint64 pattern_counter = 8;
    repeated uint64 pos_pattern_counter = 9;
}